aws/table_aws_wellarchitected_workload.go

```diff
@@ -0,0 +1,253 @@
+			},
+			{
+				Name:        "industry",
+				Description: "The industry for the workload.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getWellArchitectedWorload,
+			},
+			{
+				Name:        "industry_type",
+				Description: "The industry type for the workload.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getWellArchitectedWorload,
+			},
+			{
+				Name:        "is_review_owner_update_acknowledged",
+				Description: "Flag indicating whether the workload owner has acknowledged that the review owner field is required.",
+				Type:        proto.ColumnType_BOOL,
+				Hydrate:     getWellArchitectedWorload,
+			},
+			{
+				Name:        "notes",
+				Description: "The notes associated with the workload.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getWellArchitectedWorload,
+			},
+			{
+				Name:        "owner",
+				Description: "An AWS account ID.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "review_owner",
+				Description: "The review owner of the workload.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getWellArchitectedWorload,
+			},
+			{
+				Name:        "review_restriction_date",
+				Description: "The date and time recorded.",
+				Type:        proto.ColumnType_TIMESTAMP,
+				Hydrate:     getWellArchitectedWorload,
+			},
+			{
+				Name:        "share_invitation_id",
+				Description: "The ID assigned to the share invitation.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "updated_at",
+				Description: "The date and time recorded.",
+				Type:        proto.ColumnType_TIMESTAMP,
+			},
+			{
+				Name:        "account_ids",
+				Description: "The list of AWS account IDs associated with the workload.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getWellArchitectedWorload,
+			},
+			{
+				Name:        "aws_regions",
+				Description: "The list of AWS Regions associated with the workload, for example, us-east-2, or ca-central-1.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getWellArchitectedWorload,
+			},
+			{
+				Name:        "lenses",
+				Description: "The list of lenses associated with the workload. Each lens is identified by its LensSummary$LensAlias.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "non_aws_regions",
+				Description: "The list of non-AWS Regions associated with the workload.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getWellArchitectedWorload,
+			},
+			{
+				Name:        "pillar_priorities",
+				Description: "The priorities of the pillars, which are used to order items in the improvement plan. ",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getWellArchitectedWorload,
+			},
+			{
+				Name:        "risk_counts",
+				Description: "A map from risk names to the count of how questions have that rating.",
+				Type:        proto.ColumnType_JSON,
+			},
+			// Standard columns
```

Comment:
Add a blank line before comment
```suggestion

			// Standard columns
```